src/conduit/components/forms.cljs

```diff
@@ -62,6 +83,34 @@
                                                      :email email
                                                      :password password})))})
 
+(def article-form
+  {:fields {:title {:placeholder "Article Title"}
+            :description {:placeholder "What's this article about?"}
+            :body {:placeholder "Write your article (in markdown)"}
+            :tag {:placeholder "Enter tags"
+                  :container TagInputFieldContainer
+                  :events {:on-key-down
+                           (fn [data errors key]
```

Comment:
not sure what this code is doing

Reply:
It is trying to modify the key down behaviour:

1) to allow the creation of tags by pressing enter, then clear the input field.

2) Tags will be stored in a virtual form property named `tagList` which is used for actually sending over to the server.